Auto update failing for k9s

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the k9s package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating k9s [Current version: 0.26.3]
INFO: package being updated to 0.26.4.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 13208    0 13208    0     0  37416      0 --:--:-- --:--:-- --:--:-- 37416
100 6241k    0 6241k    0     0  8278k      0 --:--:-- --:--:-- --:--:-- 15.2M

You are about to commit these changes:

--------------------
diff --git a/packages/k9s/build.sh b/packages/k9s/build.sh
index 59e8b2fbb..41f7b7453 100644
--- a/packages/k9s/build.sh
+++ b/packages/k9s/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://k9scli.io
 TERMUX_PKG_DESCRIPTION="Kubernetes CLI To Manage Your Clusters In Style!"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="Krishna Kanhaiya @kcubeterm"
-TERMUX_PKG_VERSION="0.26.3"
+TERMUX_PKG_VERSION="0.26.4"
 TERMUX_PKG_SRCURL=https://github.com/derailed/k9s/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=30174fcfe4006230e5bbe523785537e89eb88fd38d6657b878f5b0e94635a8e3
+TERMUX_PKG_SHA256=b28f69699f6386d7b71cf09c25ecae5f988913e57d08ee7aa40e6ef73ba0325c
 TERMUX_PKG_AUTO_UPDATE=true
 
 termux_step_make() {
--------------------

k9s: update to 0.26.4


Not committing to Git!

INFO: Trying to build package.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building k9s for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  48695      0 --:--:-- --:--:-- --:--:-- 48644
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   4655      0 --:--:-- --:--:-- --:--:--  4679
gpg: Signature made Thu 22 Sep 2022 12:24:35 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  52694      0 --:--:-- --:--:-- --:--:-- 52853
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3124      0 --:--:-- --:--:-- --:--:--  3131
gpg: Signature made Sun 04 Sep 2022 06:32:25 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  42802      0 --:--:-- --:--:-- --:--:-- 42736
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3002      0 --:--:-- --:--:-- --:--:--  3007
gpg: Signature made Wed 21 Sep 2022 07:43:27 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://github.com/derailed/k9s/archive/v0.26.4.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 6241k    0 6241k    0     0  7525k      0 --:--:-- --:--:-- --:--:-- 7525k
go: downloading k8s.io/client-go v0.25.1
go: downloading k8s.io/klog/v2 v2.70.1
go: downloading github.com/rs/zerolog v1.27.0
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading k8s.io/cli-runtime v0.25.1
go: downloading github.com/go-logr/logr v1.2.3
go: downloading github.com/fvbommel/sortorder v1.0.2
go: downloading k8s.io/api v0.25.1
go: downloading k8s.io/apimachinery v0.25.1
go: downloading k8s.io/metrics v0.25.1
go: downloading github.com/adrg/xdg v0.4.0
go: downloading github.com/derailed/tview v0.7.2
go: downloading github.com/derailed/tcell/v2 v2.3.1-rc.2
go: downloading github.com/fsnotify/fsnotify v1.5.4
go: downloading github.com/sahilm/fuzzy v0.1.0
go: downloading github.com/cenkalti/backoff v2.2.1+incompatible
go: downloading github.com/fatih/color v1.13.0
go: downloading golang.design/x/clipboard v0.6.2
go: downloading sigs.k8s.io/yaml v1.3.0
go: downloading github.com/inconshreveable/mousetrap v1.0.0
go: downloading github.com/evanphx/json-patch v4.12.0+incompatible
go: downloading github.com/google/uuid v1.2.0
go: downloading k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
go: downloading github.com/gogo/protobuf v1.3.2
go: downloading github.com/google/gofuzz v1.2.0
go: downloading github.com/google/gnostic v0.5.7-v3refs
go: downloading github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7
go: downloading github.com/peterbourgon/diskv v2.0.1+incompatible
go: downloading github.com/imdario/mergo v0.3.12
go: downloading github.com/gdamore/encoding v1.0.0
go: downloading github.com/cenkalti/backoff/v4 v4.1.3
go: downloading github.com/derailed/popeye v0.10.1
go: downloading helm.sh/helm/v3 v3.9.4
go: downloading k8s.io/apiextensions-apiserver v0.25.1
go: downloading gopkg.in/inf.v0 v0.9.1
go: downloading sigs.k8s.io/structured-merge-diff/v4 v4.2.3
go: downloading k8s.io/kubectl v0.25.1
go: downloading github.com/rakyll/hey v0.1.4
go: downloading golang.org/x/image v0.0.0-20211028202545-6944b10bf410
go: downloading golang.org/x/mobile v0.0.0-20210716004757-34ab1303b554
go: downloading github.com/pkg/errors v0.9.1
go: downloading sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2
go: downloading github.com/liggitt/tabwriter v0.0.0-20181228230101-89fcab3d43de
go: downloading sigs.k8s.io/kustomize/api v0.12.1
go: downloading sigs.k8s.io/kustomize/kyaml v0.13.9
go: downloading github.com/golang/protobuf v1.5.2
go: downloading github.com/Azure/go-autorest/autorest v0.11.27
go: downloading github.com/Azure/go-autorest v14.2.0+incompatible
go: downloading github.com/Azure/go-autorest/autorest/adal v0.9.20
go: downloading golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
go: downloading github.com/google/btree v1.0.1
go: downloading google.golang.org/protobuf v1.28.0
go: downloading k8s.io/kube-openapi v0.0.0-20220803162953-67bda5d908f1
go: downloading golang.org/x/time v0.0.0-20220210224613-90d013bbcef8
go: downloading github.com/aws/aws-sdk-go v1.38.49
go: downloading github.com/prometheus/common v0.32.1
go: downloading github.com/Masterminds/semver/v3 v3.1.1
go: downloading github.com/Masterminds/sprig/v3 v3.2.2
go: downloading github.com/gosuri/uitable v0.0.4
go: downloading github.com/json-iterator/go v1.1.12
go: downloading k8s.io/component-base v0.25.1
go: downloading github.com/fatih/camelcase v1.0.0
go: downloading golang.org/x/exp v0.0.0-20200224162631-6cc2880d07d6
go: downloading github.com/Azure/go-autorest/logger v0.2.1
go: downloading github.com/Azure/go-autorest/tracing v0.6.0
go: downloading github.com/Azure/go-autorest/autorest/date v0.3.0
go: downloading github.com/golang-jwt/jwt/v4 v4.2.0
go: downloading golang.org/x/crypto v0.0.0-20220525230936-793ad666bf5e
go: downloading cloud.google.com/go v0.99.0
go: downloading google.golang.org/appengine v1.6.7
go: downloading github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
go: downloading github.com/prometheus/client_golang v1.12.1
go: downloading github.com/Masterminds/goutils v1.1.1
go: downloading github.com/huandu/xstrings v1.3.2
go: downloading github.com/mitchellh/copystructure v1.2.0
go: downloading github.com/shopspring/decimal v1.2.0
go: downloading github.com/spf13/cast v1.4.1
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369
go: downloading github.com/prometheus/client_model v0.2.0
go: downloading github.com/cyphar/filepath-securejoin v0.2.3
go: downloading github.com/xeipuuv/gojsonschema v1.2.0
go: downloading github.com/BurntSushi/toml v1.0.0
go: downloading github.com/gobwas/glob v0.2.3
go: downloading github.com/containerd/containerd v1.6.6
go: downloading github.com/opencontainers/image-spec v1.0.3-0.20211202183452-c5a74bcca799
go: downloading github.com/sirupsen/logrus v1.8.1
go: downloading oras.land/oras-go v1.2.0
go: downloading github.com/Masterminds/squirrel v1.5.3
go: downloading github.com/jmoiron/sqlx v1.3.5
go: downloading github.com/lib/pq v1.10.6
go: downloading github.com/rubenv/sql-migrate v1.1.1
go: downloading github.com/moby/spdystream v0.2.0
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/modern-go/reflect2 v1.0.2
go: downloading github.com/chai2010/gettext-go v1.0.2
go: downloading github.com/mitchellh/go-wordwrap v1.0.0
go: downloading github.com/russross/blackfriday v1.5.2
go: downloading github.com/exponent-io/jsonpath v0.0.0-20151013193312-d6023ce2651d
go: downloading github.com/go-errors/errors v1.0.1
go: downloading github.com/emicklei/go-restful/v3 v3.8.0
go: downloading github.com/go-openapi/swag v0.19.14
go: downloading github.com/go-openapi/jsonreference v0.19.5
go: downloading github.com/jmespath/go-jmespath v0.4.0
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/cespare/xxhash/v2 v2.1.2
go: downloading github.com/prometheus/procfs v0.7.3
go: downloading github.com/mitchellh/reflectwalk v1.0.2
go: downloading github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415
go: downloading github.com/asaskevich/govalidator v0.0.0-20200428143746-21a406dcc535
go: downloading k8s.io/apiserver v0.25.1
go: downloading github.com/opencontainers/go-digest v1.0.0
go: downloading github.com/docker/cli v20.10.17+incompatible
go: downloading github.com/docker/distribution v2.8.1+incompatible
go: downloading github.com/docker/docker v20.10.17+incompatible
go: downloading github.com/docker/go-connections v0.4.0
go: downloading golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
go: downloading github.com/lann/builder v0.0.0-20180802200727-47ae307949d0
go: downloading github.com/go-gorp/gorp/v3 v3.0.2
go: downloading github.com/google/go-cmp v0.5.8
go: downloading github.com/moby/term v0.0.0-20210619224110-3f7ff695adc6
go: downloading github.com/monochromegane/go-gitignore v0.0.0-20200626010858-205db1a8cc00
go: downloading github.com/xlab/treeprint v1.1.0
go: downloading github.com/mailru/easyjson v0.7.6
go: downloading github.com/PuerkitoBio/purell v1.1.1
go: downloading github.com/go-openapi/jsonpointer v0.19.5
go: downloading github.com/xeipuuv/gojsonpointer v0.0.0-20180127040702-4e3ac2762d5f
go: downloading github.com/moby/locker v1.0.1
go: downloading github.com/docker/docker-credential-helpers v0.6.4
go: downloading github.com/docker/go-units v0.4.0
go: downloading google.golang.org/grpc v1.47.0
go: downloading github.com/lann/ps v0.0.0-20150810152359-62de8c46ede0
go: downloading github.com/josharian/intern v1.0.0
go: downloading github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
go: downloading github.com/morikuni/aec v1.0.0
go: downloading github.com/klauspost/compress v1.13.6
go: downloading google.golang.org/genproto v0.0.0-20220502173005-c8bf987b8c21
go: downloading github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1
go: downloading go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5
go: downloading github.com/gorilla/mux v1.8.0
go: downloading github.com/docker/go-metrics v0.0.1
# golang.org/x/mobile/gl
In file included from /home/builder/go/pkg/mod/golang.org/x/mobile@v0.0.0-20210716004757-34ab1303b554/gl/work.go:26:
In file included from ./work.h:10:
In file included from /home/builder/.termux-build/_cache/android-r25b-api-24-v0/bin/../sysroot/usr/include/GLES2/gl2.h:37:
/home/builder/.termux-build/_cache/android-r25b-api-24-v0/bin/../sysroot/usr/include/GLES2/gl2platform.h:28:10: fatal error: 'KHR/khrplatform.h' file not found
#include <KHR/khrplatform.h>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2022-09-22 00:38:10 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/3102001157">3102001157</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/k9s/build.sh
+++ b/packages/k9s/build.sh
@@ -2,10 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://k9scli.io
 TERMUX_PKG_DESCRIPTION="Kubernetes CLI To Manage Your Clusters In Style!"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="Krishna Kanhaiya @kcubeterm"
-TERMUX_PKG_VERSION="0.26.3"
+TERMUX_PKG_VERSION="0.26.6"
 TERMUX_PKG_SRCURL=https://github.com/derailed/k9s/archive/v${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=30174fcfe4006230e5bbe523785537e89eb88fd38d6657b878f5b0e94635a8e3
+TERMUX_PKG_SHA256=afe885d074e026ac07f8ad7f2e423ab65df812a228ca6d1f96599285751c5630
 TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_BUILD_DEPEDNS="mesa"
 
 termux_step_make() {
         termux_setup_golang

